### ISSUE TITLE
Add Contact page

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/hyku#517]
Per designs at https://docs.google.com/document/d/1WZIOL8omCseweV6OSP65SkSQCxtoXbtP65UVN9doBPY/edit#heading=h.4cnihetzl8p4

**Comment by ggeisler** [comment]
I'm not sure that we want to add the Contact page to the Content Blocks configuration after all. Isn't the current Contact page a form, and in that sense different from the About and Help pages?

I know including the Contact page is in the design documented in the Google doc you're referencing, so the confusion is my fault, but I thought I had updated #440 with more recent designs (my intention for all designs is to only use the Google docs for discussing the preliminary design, and final design is always in a ticket). #440 doesn't have them so something happened there, but I'll update the other tickets with my newer designs (the same as in the Google Doc, I think, except I updated the tabs to reflect no Contact page).

**Comment by hannahfrost** [comment]
@ggeisler Are you thinking that "Contact Form" is its own facet under Configuration > Settings?

**Comment by ggeisler** [comment]
In the short term I'm thinking we just use the current Contact page we (I believe) inherit from Sufia/Hyrax. So there is no UI consideration necessary.

Longer term, we might want to add some UI configuration, which could be as simple as entering the email addresses to which the form submission should be sent, but that would probably just be a tab on another Settings page (i.e., might not warrant its own Configuration menu item). 

Of course we could completely reconsider how we want to provide a Contact page and make its configuration via the UI more powerful, but my thought is this isn't an important enough feature for MVP to do anything other than use what we inherit from Sufia/Hyrax. Happy to add it to the design needed list if you feel it should be higher-priority, though.

**Comment by hannahfrost** [comment]
Agreed, this not MVP material, I just wanted some clarity on what you were thinking.  Eventually I think a tab with these two things would be good: (1) email address(es) for where to send the form submissions and (2) customizable text (which supports HTML links) for the message at the top of the Contact form.

**Comment by hannahfrost** [comment]
@ggeisler I am renaming this ticket to "Configure contact form", flagging it as design needed, so we can make it actionable for development (though still not a top priority)

**Comment by hannahfrost** [comment]
This ticket is related: https://github.com/samvera-labs/hyrax/issues/113
